src/firebase/dynamicLinks.ts

```diff
@@ -7,24 +7,27 @@ import {
 } from 'src/config'
 import { NetworkId } from 'src/transactions/types'
 
-async function createDynamicLink(link: string) {
-  return dynamicLinks().buildShortLink({
-    link: `${WEB_LINK}${link}`,
-    domainUriPrefix: baseURI,
-    ios: {
-      appStoreId,
-      bundleId,
-    },
-    android: {
-      packageName: bundleId,
-    },
-  })
+const commentDynamicLinkParams: Omit<FirebaseDynamicLinksTypes.DynamicLinkParameters, 'link'> = {
```

Comment:
nit: The name for this shouldn't be `commonDynamicLinkParams`? I don't understand where the `comment` comes from.

Reply:
oops thank you! that was a typo 😅